### PR TITLE
Upgrade React Native library & Add README including Screenshot

### PR DESCRIPTION
**Pull request by antronic**
- For support higher React Native version.
- Add README including Screenshot

### CODE REVIEW

**Comment by hybridknight** [comment]
👍 lgtm

**Comment by phoomparin** [comment]
Looks good to me! I'm basing this fork on my refactoring attempt.

**Comment by pawin** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# AITH
+
+Let catch a turtle game!
+
+## Preview
+
+![Game Screenshot](https://raw.githubusercontent.com/DEKCOMSTORY/AITH/master/example.png)
+
+## Changelog
+- 2017/27/03 : Upgrade React-native-cli by #DEKCOMSTORY
```
Comment (dtinth) [inline README.md]: It would be great if the instructions for building the project is also added to `README.md` file. 💯

**Comment by pawin** [comment]
👍

**Comment by pariyathida** [comment]
Thanks ค่าา ><

**Pull request merged**